torch/fx/experimental/fuser.py

```diff
@@ -0,0 +1,45 @@
+from torch.fx import (
+    GraphModule,
+)
+from torch.nn.utils.fusion import fuse_conv_bn_eval
+import torch
+
+# turn foo.bar -> ['foo', 'bar']
+def _parent_name(target):
+    r = target.rsplit('.', 1)
+    if len(r) == 1:
+        return '', r[0]
+    else:
+        return r[0], r[1]
+
+# Works for length 2 patterns with 2 modules
+def matches_module_pattern(pattern, node, modules):
+    if len(node.args) == 0:
+        return False
+    nodes = [node.args[0], node]
+    for pattern_node, current_node in zip(pattern, nodes):
+        if current_node.op != 'call_module':
+            return False
+        if not isinstance(current_node, torch.fx.Node):
+            return False
```

Comment:
If `current_node` isn't a `Node`, the check `current_node.op != 'call_module'` is going to throw, isn't it? This probably needs to go before the previous check. BTW mypy should catch this stuff if the fn is type annotated